src/CMakeLists.txt

```diff
@@ -113,3 +113,13 @@ target_pc_file_generate(sigutils "Digital signal processing utility library")
 install(DIRECTORY include/sigutils DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(TARGETS sigutils DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${PROJECT_BINARY_DIR}/src/sigutils.pc DESTINATION ${CMAKE_INSTALL_PKGCONFIGDIR})
+
+# General packaging settings
+set(CPACK_PACKAGE_NAME Sigutils)
```

Comment:
Will this create a Debian package with that capital S in the name?

Reply:
Yep, I can fix that.